packages/@okta/vuepress-site/docs/guides/add-an-external-idp/main/saml2/appatidp.md

```diff
@@ -1,8 +1,8 @@
-Access the external SAML Identity Provider (IdP) you want to integrate with and gather the following information. You use this information to configure the SAML Identity Provider in Okta in the next step. Make sure the SAML IdP supports Service Provider-initiated SAML.
+Access the external SAML IdP you want to integrate with and gather the following information. Use this information to configure the SAML IdP in Okta in the next step. Ensure the SAML IdP supports Service Provider-initiated SAML.
 
-> **Note:** If you need a quick and easy SAML IdP to use for testing purposes, you can try using this [SAML Identity Provider on GitHub](https://github.com/mcguinness/saml-idp).
+> **Note:** If you need a quick and easy SAML IdP to use for testing purposes, you can use this [SAML Identity Provider on GitHub](https://github.com/mcguinness/saml-idp).
```

Comment:
Let's move this to the bottom of the snippet (L11). This is optional information that can be presented at the end.